deltachat-ios/View/ChatTitleView.swift

```diff
@@ -58,8 +58,14 @@ class ChatTitleView: UIStackView {
         titleLabel.text = title
         titleLabel.textColor = DcColors.defaultTextColor
         verifiedView.isHidden = !isVerified
-        subtitleLabel.text = subtitle
-        subtitleLabel.textColor = DcColors.defaultTextColor.withAlphaComponent(0.95)
+
+        if let subtitle = subtitle {
```

Comment:
```suggestion
        if let subtitle {
```

Reply:
thanks! i just changed what i learned today in another commit :)